actions/workflows/install_latest_revision_of_rpm_pkg.yaml

```diff
@@ -3,21 +3,44 @@ version: '1.0'
 description: Install latest revision of a specified rpm package
 input:
   - host
+  - distro
   - pkg
   - version
+
+vars:
+  - repoquery_args: ''
+
 tasks:
+  init:
+    action: core.noop
+    next:
+      - when: <% ctx().distro = 'CENTOS8' or ctx().distro = 'RHEL8' %>
+        # Run repoqery -y option in EL8 to do the following:
+        #  - import any GPG keys from new repos
+        #  - import any SSL certificates from new repos
+        publish:
+          - repoquery_args: '-y'
+        do:
+          - get_latest_revision
+      - when: <% ctx().distro = 'CENTOS7' or ctx().distro = 'RHEL7' %>
+        do:
+          - get_latest_revision
+
   get_latest_revision:
     action: core.remote
     input:
       hosts: <% ctx().host %>
-      cmd: sudo repoquery -y --show-duplicates <% ctx().pkg %>-<% ctx().version %>* | sort --version-sort | tail -n 1
```

Comment:
```
sudo repoquery -y --show-duplicates <% ctx().pkg %>-<% ctx().version %>* | sort --version-sort | tail -n 1 || sudo repoquery --show-duplicates <% ctx().pkg %>-<% ctx().version %>* | sort --version-sort | tail -n 1
```

How about a quicker and ugly hack like this?

Reply:
This is as great as it is disgusting. I like it! 😆

Reply:
I was just trying to be consistent with the way we did it in st2packages, thoughts?

Reply:
if we do it with an || statement, wouldn't stdout of that command be potentially polluted with the error from the first command and not contain only the version of the package from the second command?

Reply:
@nmaludy True, you're right :+1: